test/cucumber/step_definitions/validators_steps.js

```diff
@@ -26,20 +26,14 @@ module.exports = function() {
         body: json2
       };
 
-      return this.validate((error, result) => {
-        if (error) {
-          callback(new Error(`Got error during validation:\n${error}`));
-        }
+      try {
+        const result = this.validate();
         this.results = JSON.parse(JSON.stringify(result));
-
-        return this.validate((error, result) => {
-          if (error) {
-            callback(new Error(error));
-          }
-          this.booleanResult = result.isValid;
-          return callback();
-        });
-      });
+        this.booleanResult = result.isValid;
+        return callback();
```

Comment:
Note: try/catch around callback call will catch also errors happening in subsequent code (in the callback). Instead of fixing this I'd rather focus on upgrading cucumber in the repo though and simplifying the test steps as much as possible, with promises etc.